### PR TITLE
IntelliJ code-formatter applied to all source files; checkstyle config updates to match

### PR DESCRIPTION
**Pull request by ronreynolds**
* applied IntelliJ code-formatter to all java files (main and test)
  * only required 1 rule be disabled: `JavadocTagContinuationIndentation`
* updated checkstyle config file names to match regular names
  * and thus remove customization from `build.gradle`
* updated `java.header` to contain leading space indent on second line

### CODE REVIEW

**Comment by ronreynolds** [comment]
in order to verify a totally vanilla IntelliJ code-formatter config was applied i reverted my local config to default (i didn't think i had actually changed it) and then reran the formatter and one very noticeable change appeared: wildcard imports.  i always disable them because i think they're evil (anti-pattern?) when it comes to code readability.  somewhat conflicted whether i should leave those in or not... :-/  (checkstyle agrees with me; this rule fires ~60 times: `AvoidStarImport`)

**Review by DonoA — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -253,7 +253,6 @@ signing {
 // Checkstyle Configuration
 checkstyle {
     toolVersion = "${checkstyleVersion}"
-    configFile = file('config/checkstyle/checkstyle-custom.xml')
```
Comment (zromano) [inline build.gradle:256]: is this because we are using the default file name now?

File: config/checkstyle/suppressions.xml [hunk 2]
```diff
@@ -8,19 +8,19 @@
     <!--  Allow strings to occur multiple times without making them a variable in tests  -->
     <suppress checks="MultipleStringLiterals" files="[\\/]src[\\/]test[\\/].*"/>
     <!--  Allow underscores in test method names  -->
-    <suppress files="[\\/]src[\\/]test[\\/].*" checks="MethodName" />
+    <suppress files="[\\/]src[\\/]test[\\/].*" checks="MethodName"/>
```
Comment (zromano) [inline config/checkstyle/suppressions.xml:11]: Did you run checkstyle on our checkstyle file? 😆

File: src/main/java/com/smartsheet/api/AccessTokenExpiredException.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (C) 2024 Smartsheet
+ * Copyright (C) 2024 Smartsheet
```
Comment (zromano) [inline src/main/java/com/smartsheet/api/AccessTokenExpiredException.java:2]: Thank you for fixing this. I botched this during the 2023->2024 update 😞

**Review by priyankalakhe — DISMISSED** [review]

**Comment by zromano** [comment]
> in order to verify a totally vanilla IntelliJ code-formatter config was applied i reverted my local config to default (i didn't think i had actually changed it) and then reran the formatter and one very noticeable change appeared: wildcard imports. i always disable them because i think they're evil (anti-pattern?) when it comes to code readability. somewhat conflicted whether i should leave those in or not... :-/ (checkstyle agrees with me; this rule fires ~60 times: AvoidStarImport)

You can change this in your IntellIJ settings. I don't have a pref on this, but I know some people are anti `*` imports

**Comment by zromano** [comment]
I think you'll probably need to merge in the latest mainline to fix the failing tests

File: build.gradle (hunk 1 above)
Comment (ronreynolds) [reply build.gradle:256]: yes

File: config/checkstyle/suppressions.xml (hunk 2 above)
Comment (ronreynolds) [reply config/checkstyle/suppressions.xml:11]: probably; running the code-formatter on any file before saving is a bit of a habit.

**Comment by ronreynolds** [comment]
> I think you'll probably need to merge in the latest mainline to fix the failing tests

AFAICT it's already merged (at least according to my local git repo which i've synced with remote).  checking head commit IDs...
67dfe3a appears to be HEAD on mainline and i definitely see that commit ID on this branch.  will see if i can get this fixed today.

apparently the unit-test failure was caused by an issue between Halo and the Auth-service (either the Lua script or the service started failing in new ways, returning a 500 instead of a 401 if the Auth header didn't start with "Bearer").  spoke with IainW about this last week and it sounded like it was on the radar to be fixed.  rerunning failed stages now.  :crossed_fingers:

**Review by zromano — APPROVED** [review]

**Pull request merged**